hyperspy/tests/signal/test_1D_tools.py

```diff
@@ -48,56 +48,71 @@ def setUp(self):
         self.spectrum = s
 
     def test_estimate_shift(self):
-        s = self.spectrum
+        s = self.spectrum.deepcopy()
```

Comment:
I don't thing the `deepcopy` here (and a couple more times later) is required. The `setUp` is run before every test anyway, to ensure that identical starting conditions are created.